src/services/mfe-config-service.ts

```diff
@@ -0,0 +1,32 @@
+import * as path from 'node:path'
+import * as fs from 'node:fs'
+import { MICROFRONTENDS_FOLDER } from '../paths'
+import { MfeConfig } from '../models/mfe-config'
+
+const MFE_CONFIG_FILE_NAME = 'mfe-config.json'
+const MFE_CONFIG_INDENTATION_SPACES = 4
```

Comment:
Probably we should consider defining a global `INDENTATION_SPACES` constant for all the classes that use `JSON.stringify()`, since we have the same kind of definition in `BundleDescriptorService` and probably it will be needed somewhere else in the future.